extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -73,7 +73,16 @@ export class ResourceTypePickerDialog extends DialogBase {
 					return <azdata.RadioCard>{
 						id: resourceType.name,
 						label: resourceType.displayName,
-						icon: resourceType.icon
+						icon: resourceType.icon,
+						descriptions: [
```

Comment:
this will change how the card looks, right? can you add a screenshot?

Reply:
for the deployment dialog.

Reply:
https://github.com/microsoft/azuredatastudio/pull/12125#issuecomment-687328784

Reply:
it looks different now, previously the label is center aligned vertically

Reply:
![image](https://user-images.githubusercontent.com/1465785/92283759-76551480-eeb5-11ea-8933-9c45ae350fc2.png)
This is what it looks like on insiders 

![image](https://user-images.githubusercontent.com/1465785/92283776-7d7c2280-eeb5-11ea-916c-8e4cd5ff7396.png)
this is what it looks like with this PR

Reply:
I could also use flexbox to align them to the center